lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -35,6 +37,25 @@ def ask_questions
       confirm
     end
 
+    def choose_disk
+      @disk = ask_for_disk("database disk")
```

Comment:
I don't think this really needs a method. You can just do this in `#ask_questions`.